infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/PodTerminationGracePeriodProvisioner.java

```diff
@@ -31,6 +36,14 @@
  */
 public class PodTerminationGracePeriodProvisioner implements ConfigurationProvisioner {
   private final long graceTerminationPeriodSec;
+  /**
+   * This value will activate if workspace configured to use Async Storage. We can't set default
+   * grace termination period because we need to give some time on workspace stop action for backup
+   * changes to the persistent storage. At the moment no way to predict this time because it depends
+   * on amount of files, size of files and network ability. This is some empirical number of seconds
+   * which should be enough for most projects.
+   */
+  private final long graceTerminationPeriodAsyncPvc = 60;
```

Comment:
should be `static`. Also it's a Java convention to write constant names in capitals.

Reply:
fixed